base/server/python/pki/server/deployment/scriptlets/fapolicy_setup.py

```diff
@@ -0,0 +1,93 @@
+# Authors:
+#     Marco Fargetta <mfargett@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright (C) 2023 Red Hat, Inc.
+# All rights reserved.
+#
+
+from __future__ import absolute_import
+import logging
+import os
+import shutil
+import subprocess
+
+# PKI Deployment Imports
+from .. import pkiconfig as config
+from .. import pkiscriptlet
+
+fapolicy_rules_path = '/etc/fapolicyd/rules.d/'
+
+fapolicy_rule_file = None
+if os.path.exists(fapolicy_rules_path):
+    fapolicy_rule_file = os.path.join(fapolicy_rules_path, '61_pki.rules')
```

Comment:
Looks like existing files use dashes (`-`) instead of underscores (`_`), so we probably should follow the convention.

Reply:
The example here uses `80-myapps.rules` so maybe we want to use `80` instead of `61`?
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_blocking-and-allowing-applications-using-fapolicyd_security-hardening#proc_adding-custom-allow-and-deny-rules-for-fapolicyd_assembly_blocking-and-allowing-applications-using-fapolicyd

Reply:
> we want to use `80` instead of `61`?

Actually, the rules denying the execution are in the file starting with 70:

`70 - trusted language rules `

therefore we have to use a smaller number because rules are applied in order.

Reply:
Sure, that's fine.